### PR TITLE
[Doc] add note for dynamic config and update hive catalog

### PR DESCRIPTION
**Pull request by evelynzhaojie**
Fixes #issue

## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

## Checklist:
- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.1
  - [x] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

**Review by hellolilyliuyi — DISMISSED** [review]

File: docs/administration/Configuration.md [hunk 1]
```diff
@@ -36,6 +36,10 @@ You can configure or modify the settings of FE dynamic parameters using [ADMIN S
 ADMIN SET FRONTEND CONFIG ("key" = "value");
 ```
 
+> **NOTE**
+>
+> The configurations will be restored to the default values in the `fe.conf` file after the FE reboots. Therefore, we recommend that you also modify the configuration items in `fe.conf` to prevent the loss of modifications.
```
Comment (EsoragotoSpirit) [inline docs/administration/Configuration.md:41]: restarts？

File: docs/sql-reference/sql-statements/Administration/ADMIN SET CONFIG.md [hunk 2]
```diff
@@ -2,9 +2,9 @@
 
 ## Description
 
-This statement is used to set configuration items for the cluster (Currently, only FE configuration items can be set using this command). You can view these configuration items using the ADMIN SHOW FRONTEND CONFIG command.
+This statement is used to set configuration items for the cluster (Currently, only FE configuration items can be set using this command). You can view these configuration items using the [ADMIN SHOW FRONTEND CONFIG](ADMIN%20SET%20CONFIG.md) command.
 
-The configurations will be restored to the default values in `fe.cof` after the FE reboots. Therefore, we recommended that you also modify the configuration items in `fe.conf` to prevent the loss of modifications.
+The configurations will be restored to the default values in the `fe.conf` file after the FE reboots. Therefore, we recommend that you also modify the configuration items in `fe.conf` to prevent the loss of modifications.
```
Comment (EsoragotoSpirit) [inline docs/sql-reference/sql-statements/Administration/ADMIN SET CONFIG.md:7]: restarts？

File: docs/sql-reference/sql-statements/Administration/ADMIN SET CONFIG.md [hunk 3]
```diff
@@ -2,9 +2,9 @@
 
 ## Description
 
-This statement is used to set configuration items for the cluster (Currently, only FE configuration items can be set using this command). You can view these configuration items using the ADMIN SHOW FRONTEND CONFIG command.
+This statement is used to set configuration items for the cluster (Currently, only FE configuration items can be set using this command). You can view these configuration items using the [ADMIN SHOW FRONTEND CONFIG](ADMIN%20SET%20CONFIG.md) command.
```
Comment (EsoragotoSpirit) [inline docs/sql-reference/sql-statements/Administration/ADMIN SET CONFIG.md:5]: only FE **dynamic** configuration items？

**Review by EsoragotoSpirit — APPROVED** [review]

**Review by hellolilyliuyi — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.0

**Pull request merged**